client/src/context/useAuthContext.tsx

```diff
@@ -8,12 +8,14 @@ import logoutAPI from '../helpers/APICalls/logout';
 interface IAuthContext {
   loggedInUser: User | null | undefined;
   updateLoginContext: (data: AuthApiDataSuccess) => void;
+  updateLoggedInUser: (data: User) => void;
```

Comment:
Maybe try using the updateLoginContext instead of creating a new function to avoid any confusion in the future?

Reply:
I debated this a bit when I did it before.  I feel more and more we should have a user context and an auth context.  In any case I made this update and removed the updateLoggedInUser function throughout.